clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -44,6 +44,13 @@ from functools import wraps
 import google.protobuf.text_format
 from google.rpc import status_pb2, code_pb2
 
+try:
```

Comment:
Good idea to put this inside a try/catch block instead of assuming default installed packages. Throwing an exception and continuing is a better approach, if the idpf packages are not installed